libraries/AP_SerialLED/AP_SerialLED.h

```diff
@@ -32,15 +33,15 @@
 class AP_SerialLED {
 public:
     // set number of LEDs per pin
-    bool set_num_neopixel(uint8_t chan, uint8_t num_leds);
+    bool set_num_neopixel(uint8_t chan, uint8_t num_leds, bool rgb = false);
```

Comment:
I would prefer a new method so we can expose it to scripting. We don't support optional arguments for auto generated bindings. `set_num_neopixel_rgb` sort of thing.

Reply:
the issue is the setup is done at the same time, do the bindings support overloading? Or do you mean something like ```set_num_neopixel_rgb()```

Reply:
I was just thinking a new method name so we then have three, our led scripts typically looks like this:

https://github.com/ArduPilot/ardupilot/blob/1641aa42ec37884caf204195c5d97eb97663e334/libraries/AP_Scripting/examples/LED_roll.lua#L29-L31

Where you would just on-comment the type you have.

Reply:
Fixed